StatsigRedis/StatsigRedis.csproj

```diff
@@ -9,8 +9,9 @@
         <Version>1.0.0</Version>
         <Authors>Statsig Inc.</Authors>
         <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+        <RepositoryUrl>https://github.com/statsig-io/dotnet-sdk.git</RepositoryUrl>
         <Owners>Statsig Inc.,</Owners>
-        <PackageLicenseFile>LICENSE</PackageLicenseFile>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Adding the license expression is great, but this repo and all of our sdks are under the ISC license

https://github.com/statsig-io/dotnet-sdk/blob/main/LICENSE

Reply:
All fixed